src/__tests__/type.js

```diff
@@ -232,6 +232,7 @@ test('should delay the typing when opts.delay is not 0', async () => {
 
   const text = 'Hello, world!'
   const delay = 10
+  // eslint-disable-next-line testing-library/no-await-sync-events
   await userEvent.type(element, text, {delay})
```

Comment:
Is the plugin not picking up the overload anymore ? :disappointed: 

This might be just another reason for us to always return a Promise on `userEvent.type` in the next major.

Reply:
It seems like it. I've opened https://github.com/testing-library/eslint-plugin-testing-library/issues/403 so we can discuss it there.

Would you still want to merge this in the meantime or would you rather wait for that to be fixed?